src/main/java/org/broadinstitute/hellbender/utils/config/CustomBooleanConverter.java

```diff
@@ -0,0 +1,16 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import org.aeonbits.owner.Converter;
+
+import java.lang.reflect.Method;
+
+/**
+ * Created by jonn on 7/21/17.
+ */
+public class CustomBooleanConverter implements Converter<Boolean> {
+
+    @Override
+    public Boolean convert(Method method, String input) {
```

Comment:
Ah.  I'm not 100% sure - it's an `OWNER` construct to make it sync up with the deserializer.  It's not documented very well on the website.